src/transformers/tsx.ts

```diff
@@ -15,26 +15,45 @@ export default function transformTsx(code: string, node: Node, cssRoot: Root): T
       s.overwrite(match.index, match.index + match[1].length + 2,`= \`${utilityClasses} \${${match[1]}}\``)
     }
     return {
-      code: s.toString().replace(/\\:/g, '\\\\:')
+      code: s.toString().replace(/\\/g, '\\\\')
```

Comment:
I think you actually need to double escape all backlashes to provide proper support for classes starting with a number e.g. `2xl:w-1/2`.